R/multi_color_rug.R

```diff
@@ -1,6 +1,7 @@
 ## multicolor rug function
 multi_color_rug <- function(den, col="YlOrRd", ticksize=0.03, side=1, lwd=0.5) 
 {
+    if(0 >= side | side > 5 ) {stop("Input value for side must be an intereger between 1 and 4.")}
```

Comment:
Thanks! This is great. A couple suggestions:
1. I think it should be "side > 4"
2. Also integer is misspelled. 
3. You might want to check that "side" is an integer as well.